production/scrapers/arkansas_psychiatric.R

```diff
@@ -0,0 +1,68 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+arkansas_psychiatric_pull <- function(x){
+    get_src_by_attr(x, "a", attr = "href", attr_regex = "(?i)congregate") %>%
+        first()
+}
+
+arkansas_psychiatric_restruct <- function(x){
+    stop_defunct_scraper("https://www.healthy.arkansas.gov/programs-services/topics/covid-19-reports")
+}
+
+arkansas_psychiatric_extract <- function(x){
+    x
+}
+
+#' Scraper class for general North Carolina COVID data
+#' 
+#' @name arkansas_psychiatric_scraper
+#' @description NC has data for many congregate settings is compiled by DHHS.
```

Comment:
Update to Arkansas instead of NC! Also, is this called psychiatric because one of the facilities is? Since it has DOCs (and I guess jails?) too, maybe it should be named more generally?